CMakeLists.txt

```diff
@@ -54,6 +54,18 @@ if(PY4DGEO_WITH_OPENMP)
     target_link_libraries(py4dgeo PUBLIC OpenMP::OpenMP_CXX)
     target_compile_definitions(py4dgeo PUBLIC PY4DGEO_WITH_OPENMP
                                               EIGEN_DONT_PARALLELIZE)
+    if(MSVC)
+      include(CheckCXXCompilerFlag)
+      check_cxx_compiler_flag("/openmp:llvm" MSVC_SUPPORTS_OPENMP_LLVM)
+      if(MSVC_SUPPORTS_OPENMP_LLVM)
+        target_compile_options(py4dgeo PRIVATE "/openmp:llvm")
```

Comment:
As we are both linking the target `OpenMP::OpenMP_CXX` *and* adding `/openmp:llvm` - are we entirely sure that having both flags (`/openmp /openmp:llvm`) does what we want? Looking into the CMake docs, it seems that CMake 3.30 added controls for the MSVC OpenMP runtime: https://cmake.org/cmake/help/latest/module/FindOpenMP.html#input-variables